muni_portal/settings.py

```diff
@@ -198,8 +199,16 @@
 WAGTAILAPI_BASE_URL = env.str("WAGTAILAPI_BASE_URL", None)
 FRONTEND_BASE_URL = env.str("FRONTEND_BASE_URL", None)
 
-CORS_URLS_REGEX = r"^/api/.*$"
-CORS_ALLOW_ALL_ORIGINS = True
+CORS_URLS_REGEX = r"^/api/(accounts|token)/.*$"
+CORS_ALLOWED_ORIGIN_REGEXES = [re.compile(x) for x in env.list("CORS_ALLOWED_ORIGIN_REGEXES", [])]
+CORS_ALLOW_HEADERS = [
+    "accept",
```

Comment:
please use corsheaders.defaults.default_headers as per the docs instead of hardcoding these